### ISSUE TITLE
Minor logic error in lambda timeout (probably not noticeable)

### ISSUE THREAD

**Issue by matthewmueller** [apex/up#599]
## Prerequisites

* [x] I am running the latest version. (`up upgrade`)
* [x] I searched to see if the issue already exists.
* [x] I inspected the verbose debug output with the `-v, --verbose` flag.
* [x] Are you an Up Pro subscriber?

## Description

During my foray into the Up code base, I noticed that the timeout logic seemed a bit off: https://github.com/apex/up/blob/master/platform/lambda/lambda.go#L540

Not a big deal, but I think this logic will add 3 seconds of timeout no matter what. I believe it should be something like this: 

```
Timeout:      aws.Int64(int64(math.Max(float64(in.Timeout), 3)))
```

Basically `[3, in.Timeout]`

## Love Up?

Yes ❤️

**Comment by tj** [comment]
Ahh this is somewhat arbitrary, and should have a comment, but that additional 3 seconds is to allow the proxy in Go-land to timeout at `p.config.Proxy.Timeout` so we have full control over the response

**Comment by matthewmueller** [comment]
Ahh nevermind then!
